dts/common/nordic/nrf54h20.dtsi

```diff
@@ -987,6 +987,19 @@
 				status = "disabled";
 				interrupts = <470 NRF_DEFAULT_IRQ_PRIORITY>;
 			};
+
+			exmif: spi@95000 {
+				compatible = "nordic,nrf-exmif", "snps,designware-spi";
```

Comment:
I don't think that "snps,designware-spi" should be listed here. It is not actually possible to use this peripheral that way without Nordic specific stuff.

Reply:
Removing the compatible results in spi_dw.c not being compiled and `DT_INST_FOREACH_STATUS_OKAY` not picking up the node.